staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer.go

```diff
@@ -133,7 +133,9 @@ func (g *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, svc
 	if err != nil {
 		return nil, err
 	}
-
+	if err := checkMixedProtocol(svc.Spec.Ports); err != nil {
+		return nil, err
```

Comment:
We should emit an event or something to indicate this LB couldn't be created due to mixed protocol (prior to allowing mixed protocol this was done in Service validation)

Reply:
added a log warning and an event